config/initializers/decidim.rb

```diff
@@ -5,7 +5,7 @@
   config.application_name = "My Application Name"
 
   # The email that will be used as sender in all emails from Decidim
-  config.mailer_sender = "change-me@example.org"
+  config.mailer_sender = "presupuestoparticipativo@monterrey.gob.mx"
```

Comment:
Podemos dejar esta configuración en una variable de ambiente?
Asi no dejamos aqui informacion sensible 😄 
```suggestion
  config.mailer_sender = ENV.fetch("MAILER_SENDER")
```

Reply:
Lo único que le faltaría a eso es que si no se configura la variable de entorno... no hay sender, ¿no?

Reply:
Correcto, pero la forma en la que hacemos `fetch` obligaria a que la env exista. Como no hay un valor por defecto, al iniciar la app, si la env no esta presente, simplemente no correria 😄